com.unity.cinemachine/Documentation~/TableOfContents.md

```diff
@@ -34,9 +34,7 @@
                 * [Cinemachine Confiner2D](CinemachineConfiner2D.md)
             * [Group Framing extension](CinemachineGroupFraming.md)
             * [Follow Zoom extension](CinemachineFollowZoom.md)
-            * [Pixel Perfect extension](CinemachinePixelPerfect.md)
             * [Post Processing extension](CinemachinePostProcessing.md)
-            * [Volume Settings extension](CinemachineVolumeSettings.md)
```

Comment:
You need to put these back, they still exist.  PixelPerfect is only available  in URP projects, and VolumeSettings is only available in URP and HDRP projects.